core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/AggregatedWorkspaceServiceSpec.scala

```diff
@@ -31,7 +31,7 @@ import scala.language.postfixOps
 
 class AggregatedWorkspaceServiceSpec extends AnyFlatSpec with MockitoTestUtils {
 
-  private val rawlsWorkspace = Workspace(
+  private val legacyRawlsWorkspace = Workspace(
```

Comment:
Just renamed this and `mcRawlsWorkspace` to clarify that these are the Rawls side of things (as opposed to being WSM workspaces, which are individually mocked in the specs).